doc/radosgw/group.rst

```diff
@@ -0,0 +1,100 @@
+===============
+Groups
+===============
+
```

Comment:
would be nice to have a `versionadded` tag here (for Luminous i guess, unless this doesn't make it?)

for example, multisite.rst has `.. versionadded:: Jewel` which renders as
> _New in version Jewel._